src/random.h

```diff
@@ -351,19 +363,19 @@ class FastRandomContext : public RandomMixin<FastRandomContext>
         return ReadLE64(UCharCast(buf.data()));
     }
 
-    /** Fill a byte Span with random bytes. */
+    /** Fill a byte Span with random bytes. This overrides the RandomMixin version. */
```

Comment:
This is kinda sneaky and surprising (to me at least) to see with CRTP.

It would be nice to specify as part of the concept which functions are _not_ allowed to be overridden, but sadly 
[std::experimental::detected_t](https://en.cppreference.com/w/cpp/experimental/is_detected) is.. well.. experimental.